src/components/TeacherComponents/SpecialistHeaderListing/SpecialistHeaderListing.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from "react";
+
+import { Breadcrumb } from "@/components/SharedComponents/Breadcrumbs";
+import { LessonHeaderWrapper } from "@/components/TeacherComponents/LessonHeaderWrapper";
+import SubjectIconBrushBorders from "@/components/TeacherComponents/SubjectIconBrushBorders";
+import { Heading, P, Span } from "@/components/SharedComponents/Typography";
+import Flex from "@/components/SharedComponents/Flex";
+import { OakColorName } from "@/styles/theme";
+import Grid, { GridArea } from "@/components/SharedComponents/Grid";
+
+/**
+ * This is a header for the specialist listing pages (programme).
+ *
+ */
+
+export type SpecialistHeaderListingProps = {
+  breadcrumbs: Breadcrumb[];
+  background?: OakColorName;
+  subjectTitle: string;
+  subjectSlug: string;
+  subjectIconBackgroundColor?: OakColorName;
+  title: string;
+  hasCurriculumDownload?: boolean;
+  description: string;
+};
+
+const SpecialistHeaderListing: FC<SpecialistHeaderListingProps> = (props) => {
+  const {
+    subjectSlug,
+    title,
+    subjectTitle,
+    subjectIconBackgroundColor = "white",
+    breadcrumbs,
+    background = "aqua",
+    description,
+  } = props;
+
+  return (
+    <LessonHeaderWrapper breadcrumbs={breadcrumbs} background={background}>
+      <Grid>
+        <GridArea $colSpan={[12, 6]}>
```

Comment:
what's the reason for using a grid where there's only one item?

Reply:
There will be video, but the ticket said not to include this yet.

Reply:
Aha I see, generally I'd advise against doing things we don't need now because we might need them in the future - often that thing doesn't end up happening or things have changed so we need to change it anyway! not a big deal in this case